### ISSUE TITLE
[ENHANCEMENT] Calculation of confidence interval in autocorrelation plot

### ISSUE THREAD

**Issue by tdmeij** [pastas/pastas#434]
**Proposed enhancement**
Given a Pandas time series sr with noise values from a Pastas model,the function ps.stats.acf calculates the autocorrelation of residuals.
For example:
```
r = ps.stats.acf(sr, lags=365, bin_method='rectangle', bin_width=0.5, max_gap=1000,
        min_obs=20, full_output=True, alpha=0.05)
```

Variable r is a pandas DataFrame with columns 'acf', 'stderr' and 'n'. The column name 'stderr' confuses me because in module plots the function plots.acf calculates confidence intervals for autocorrelation as:
```
conf = r.stderr.values
ax.fill_between(r.index.days, conf, -conf, alpha=0.3)
```

Usually confidence intervals for autocrorrelation are plotted at 2*std, assuming normal distribution of noise values.

**Expected behavior**
For consistency with literature I would suggest either renaming the column 'std' to 'conf' in the result of stats.acf, or calculating the confidence interval in plot.acf as 2*std (and adjust calculation of the std in stats.acf accordingly).

**Comment by raoulcollenteur** [comment]
Hi @tdmeij, thanks for opening the issue and sorry for the late reply. I looked into this just now and agree this should be more consistent. I will look into this later this week and decide whether to 

a. change the return of ps.stats.acf to stderr instead of conf, or
b. change the plot function.

Cheers,
Raoul

**Comment by raoulcollenteur** [comment]
I will change the return of the CCF and ACf to return a DataFrame with the column name "conf". This way, the keyword `alpha` can still be used throughout the methods (plots, ccs, acf).

**Comment by raoulcollenteur** [comment]
Issue solved as per PR #465. Thanks for opening the Issue @tdmeij , keep 'em coming! :)

Cheers, Raoul
